includes/Modules/Analytics.php

```diff
@@ -595,30 +563,46 @@ protected function get_datapoint_services() {
 	 * @return array Map of datapoints to their definitions.
 	 */
 	protected function get_datapoint_definitions() {
-		$map = parent::get_datapoint_definitions();
+		$map = array(
+			'GET:connection'                   => array( 'service' => '' ),
+			'GET:account-id'                   => array( 'service' => '' ),
+			'GET:property-id'                  => array( 'service' => '' ),
+			'GET:profile-id'                   => array( 'service' => '' ),
+			'GET:internal-web-property-id'     => array( 'service' => '' ),
+			'GET:use-snippet'                  => array( 'service' => '' ),
+			'GET:tracking-disabled'            => array( 'service' => '' ),
+			'GET:anonymize-ip'                 => array( 'service' => '' ),
+			'GET:goals'                        => array( 'service' => 'analytics' ),
+			'GET:accounts-properties-profiles' => array( 'service' => 'analytics' ),
+			'GET:properties-profiles'          => array( 'service' => 'analytics' ),
+			'GET:profiles'                     => array( 'service' => 'analytics' ),
+			'GET:tag-permission'               => array( 'service' => '' ),
+			'GET:report'                       => array( 'service' => 'analyticsreporting' ),
+			'POST:connection'                  => array( 'service' => '' ),
+			'POST:account-id'                  => array( 'service' => '' ),
+			'POST:property-id'                 => array( 'service' => '' ),
+			'POST:profile-id'                  => array( 'service' => '' ),
+			'POST:internal-web-property-id'    => array( 'service' => '' ),
+			'POST:use-snippet'                 => array( 'service' => '' ),
+			'POST:tracking-disabled'           => array( 'service' => '' ),
+		);
 
-		$map['POST:create-account-ticket'] = array_merge(
-			$map['POST:create-account-ticket'],
-			array(
-				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.provision' ),
-				'request_scopes_message' => __( 'You’ll need to grant Site Kit permission to create a new Analytics account on your behalf.', 'google-site-kit' ),
-			)
+		$map['POST:create-account-ticket'] = array(
```

Comment:
Similar to my previous comment about sorting, let's always `return array(` in this method now that we aren't calling the parent and move these separate definitions to be inline in the main array in their respective places regardless if they have additional keys or not. Definitions with the extra keys can span multiple lines, the same as it is done here 👍

Reply:
Updated.